src/docfx/build/xref/XrefResolver.cs

```diff
@@ -32,6 +33,7 @@ internal class XrefResolver
 
             _dependencyMapBuilder = dependencyMapBuilder;
             _fileLinkMapBuilder = fileLinkMapBuilder;
+            _xrefHostName = context.Config.XrefHostName;
```

Comment:
fallback to host name in `base_url` in case this config is not set?

Reply:
sounds good